Core/Jira/ServiceFacadeImplementations/JiraIssueService.cs

```diff
@@ -96,4 +96,25 @@ public IReadOnlyList<JiraToBeMovedIssue> FindAllClosedIssues (string versionId)
 
     return filteredIssues.ToList();
   }
+
+  public void AddFixVersionToIssues (IEnumerable<JiraToBeMovedIssue> issues, string versionId)
+  {
+    foreach (var issue in issues)
+    {
+      var resource = $"issue/{issue.ID}";
+      var request = _jiraRestClientProvider.GetJiraRestClient().CreateRestRequest(resource, Method.PUT);
+
+      if (issue.Fields == null)
+        throw new InvalidOperationException($"Could not get fields from issue '{issue}");
+
+      var newFixVersions = issue.Fields.FixVersions;
+      newFixVersions.Add(new JiraVersion { ID = versionId });
+
+      var body = new { fields = new { fixVersions = newFixVersions.Select(v => new { id = v.ID }) } };
```

Comment:
this is actually code duplication. a project helper method could be extracted. Perhaps...
 var body = new { fields =new {GetDtoFrom(newFixVersions) }};